pkg/controllers/endpoints/endpoints_lb_controller.go

```diff
@@ -0,0 +1,4 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
```

Comment:
Is there any plan to update all license files in this repo? I saw existing files are still using VMware license.

Reply:
@lxiaopei